### PR TITLE
Readme policing

### PR DESCRIPTION
**Pull request by eugenserbanescu**
Readme police goes weee wooo weee wooo

### CODE REVIEW

File: y2020/ex16/README.md [hunk 1]
```diff
@@ -1,64 +1,65 @@
...
 
-The rules for ticket fields specify a list of fields that exist somewhere on the ticket and the valid ranges of values for each field. For example, a rule like class: 1-3 or 5-7 means that one of the fields in every ticket is named class and can be any value in the ranges 1-3 or 5-7 (inclusive, such that 3 and 5 are both valid in this field, but 4 is not).
+The rules for ticket fields specify a list of fields that exist somewhere on the ticket and the valid ranges of values for each field. For example, a rule like class: `1-3` or `5-7` means that one of the fields in every ticket is named class and can be any value in the ranges `1-3` or `5-7` (inclusive, such that `3` and `5` are both valid in this field, but `4` is not).
 
 Each ticket is represented by a single line of comma-separated values. The values are the numbers on the ticket in the order they appear; every ticket has the same format. For example, consider this ticket:
 
-.--------------------------------------------------------.
```
Comment (malagolichiara) [inline y2020/ex16/README.md:12]: I would have kept the dashes lines

Comment (eugenserbanescu) [reply y2020/ex16/README.md:12]: Not sure if it's worth making this an actual table .. the inputs don't look like this, so this representation is more for people without imagination I guess?

Comment (malagolichiara) [reply y2020/ex16/README.md:12]: no,no, I mean inside the triple backquote sections

Comment (eugenserbanescu) [reply y2020/ex16/README.md:12]: Yep. It had that line to make it look like an actual paper ticket or something. But I'm thinking whether it's worth keeping it like that, or just making it look like the inputs. But it's prolly best to leave it as it is (even if it doesn't provide value to the problem)

**Review by malagolichiara — APPROVED** [review]

**Review by lmammino — APPROVED** [review]
Nice policing 🚔

**Pull request merged**